tests/utils/test_nvfp4_per_token_quantize_large_m.py

```
@@ -0,0 +1,84 @@
+    )
+
+    device = torch.device("cuda")
+    # Fill each row with a per-row constant so the returned per-token scale
+    # (row amax with a global scale of 1.0) identifies the row that was
+    # actually read. A wrapped row address either faults or fails the
+    # exact-scale comparison below.
+    row_values = (1.0 + (torch.arange(rows, device=device) % 7)).float()
+    x = row_values.to(torch.bfloat16).unsqueeze(1).expand(rows, K).contiguous()
+    global_scale_inv = torch.ones(1, dtype=torch.float32, device=device)
+
+    fp4, sf, per_token_scale = nvfp4_quantize_per_token_cute_dsl(
```

Ruff (0.16.0)

[warning] Unpacked variable `sf` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)